client/src/components/CenterpieceDropDown.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+
+
+const CenterpieceDropDown = (props) => {
+  console.log('this is friends prop: ', props.friends)
```

Comment:
You might want to remove a lot of this console logging to keep things relatively clean